src/current/cockroachcloud/basic-cluster-management.md

```diff
@@ -35,11 +35,13 @@ The monthly cost estimate is calculated using simple extrapolation that assumes
 
 ## Edit cluster capacity
 
-On the **Overview** page, you can edit your resource limits. Changes apply to the current and future billing cycles. For more details, refer to [Plan a CockroachDB {{ site.data.products.basic }} cluster]({% link cockroachcloud/plan-your-cluster-basic.md %}).
+The price-performance characteristics of CockroachDB {{ site.data.products.basic }} vary significantly depending on the workload. It can be difficult to estimate a workload's compute requirements in advance. To increase or decrease a cluster's capacity:
```

Comment:
@mdlinville there is a mismatch here. Cluster capacity is relevant only to Standard, not Basic.

Reply:
Basic is on-demand so there's no capacity setting.